src/python/aqueduct_executor/operators/connectors/tabular/relational.py

```diff
@@ -24,7 +26,15 @@ def discover(self) -> List[str]:
         return inspect(self.engine).get_table_names()
 
     def extract(self, params: extract.RelationalParams) -> pd.DataFrame:
-        df = pd.read_sql(params.query, con=self.engine)
+        query = params.query
+        matches = re.findall(r"{{[\s+]*\w+[\s+]*}}", query)
```

Comment:
Can we make this a constant up top and document it with comment indicating what it does.